lib/commands/install.js

```diff
@@ -20,18 +21,19 @@ module.exports = Command.extend({
         'Use this option to force the usage of a specific package manager. ' +
         'By default, ember-cli will try to detect the right package manager ' +
         'from any lockfiles that exist in your project.',
-      aliases: [{ yarn: 'yarn' }, { pnpm: 'pnpm' }],
+      aliases: [{ npm: 'npm' }, { pnpm: 'pnpm' }, { yarn: 'yarn' }],
     },
   ],
 
   anonymousOptions: ['<addon-name>'],
 
-  run(commandOptions, addonNames) {
+  async run(commandOptions, addonNames) {
     if (!addonNames.length) {
-      let msg = 'The `install` command must take an argument with the name';
-      msg += ' of an ember-cli addon. For installing all npm ';
-      msg += 'dependencies you can run `npm install`.';
```

Comment:
`npm install` was still hardcoded here. Should now use the correct package manager.